### PR TITLE
Exempt state for MHR Information

### PR DESCRIPTION
**Pull request by dimak1**
*Issue #:* 
- bcgov/entity#17422

*Description of changes:*
- MHR Info page in Exempt state (any changes restricted)
- Other updates as per design req's
- Unit tests
- Small cleanups

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the PPR license (Apache 2.0).

### CODE REVIEW

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://bcregistry-assets-dev--pr-1591-zmzozvg0.web.app

**Review by doug-lovett — APPROVED** [review]
Looks great.

File: ppr-ui/src/components/unitNotes/UnitNotePanels.vue [hunk 1]
```diff
@@ -113,7 +122,11 @@ export default defineComponent({
 
     const localState = reactive({
       activePanels: [],
-      panelUnitNotes: createPanelUnitNotes(props.unitNotes)
+      panelUnitNotes: createPanelUnitNotes(props.unitNotes),
+      addUnitNoteDropdown: computed((): UnitNoteDocTypes[] => {
+        const dropdown = isRoleStaffReg ? ResidentialExemptionStaffDropDown : ResidentialExemptionQSDropDown
```
Comment (dimak1) [inline ppr-ui/src/components/unitNotes/UnitNotePanels.vue:127]: @cameron-eyds @doug-lovett for awareness, I'm using this reg staff role (ppr_staff), but across the app we also do use 'staff'.

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://bcregistry-assets-dev--pr-1591-zmzozvg0.web.app

**Review by cameron-eyds — APPROVED** [review]
Looks good!

File: ppr-ui/src/components/unitNotes/UnitNotePanels.vue (hunk 1 above)
Comment (doug-lovett) [reply ppr-ui/src/components/unitNotes/UnitNotePanels.vue:127]: Thanks Dima, the API looks at ppr_staff, not staff, when checking endpoint access.

**Review by cameron-eyds — APPROVED** [review]
Looks good!

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://bcregistry-assets-dev--pr-1591-zmzozvg0.web.app

**Pull request merged**